src/collections/Table/TableHeaderCell.js

```diff
@@ -1,16 +1,31 @@
-import React from 'react'
-
-import { META } from '../../lib'
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import { META, useValueAndKey } from '../../lib'
 import TableCell from './TableCell'
 
 function TableHeaderCell(props) {
-  return <TableCell {...props} />
+  const { className, sorted, ...rest } = props
+  const classes = cx(
+    useValueAndKey(sorted, 'sorted'),
+    className
+  )
+  return <TableCell {...rest} className={classes} />
```

Comment:
We use `getUnhandledProps` function instead of destructing. Let's make some refactor there:

```jsx
const { className, sorted } = props
const classes = cx(
  useValueAndKey(sorted, 'sorted'),
  className,
)
const rest = getUnhandledProps(TableHeaderCell, props)

return <TableCell {...rest} className={classes} />
```